coruja/script.js

```diff
@@ -0,0 +1,87 @@
+function createProductImageElement(imageSource) {
+  const img = document.createElement('img');
+  img.className = 'item__image';
+  img.src = imageSource;
+  return img;
+}
+
+function createCustomElement(element, className, innerText) {
+  const e = document.createElement(element);
+  e.className = className;
+  e.innerText = innerText;
+  return e;
+}
+
+function createProductItemElement({ sku, name, image }) {
+  const section = document.createElement('section');
+  section.className = 'item';
+  section.appendChild(createCustomElement('span', 'item__sku', sku));
+  section.appendChild(createCustomElement('span', 'item__title', name));
+  section.appendChild(createProductImageElement(image));
+  section.appendChild(createCustomElement('button', 'item__add', 'Adicionar ao carrinho!'));
+  return section;
+}
+
+function gerateLocalStoragePosition() {
+  let localStorageLength = localStorage.length;
+  while (localStorage[localStorageLength]) {
+    localStorageLength += 1;
+  }
+  return localStorageLength;
+}
+
+const setKeyStorageCar = ({ sku }) => {
+  localStorage[gerateLocalStoragePosition()] = sku;
+};
+
+function API_KEY() {
+  return localStorage.getItem('API_KEY');
+}
+
+function saveNameInPage() {
+  const nomeDeEntrada = document.getElementsByClassName('input-name')[0];
+  nomeDeEntrada.addEventListener('keyup', function addName() {
+    sessionStorage.setItem('Nome', this.value);
+  });
+}
+
+function cartItemClickListener(event) {
+  event.target.remove();
```

Comment:
Cara, as vezes nessa sintaxe o Chrome buga. pra ficar mais didático, te aconselho a criar uma variavel que recebe event.target, e depois chamar o remove.
```suggestion
const eventLocation = event.target
  eventLocation.remove();
```

Reply:
Boa.